### PR TITLE
Create META.json

### PR DESCRIPTION
**Pull request by PerikAdjust**
@za-arthur I put you in maintainer with rapimo, I don't know if it's the good things to do.

### CODE REVIEW

**Comment by za-arthur** [comment]
You can remove `maintainer` variable. I think we don't have dedicated maintainers on this. In case of issues one can just create an issue on the repo.

File: META.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+   "name": "lock_pool",
+   "abstract": "To wait on a lock pool",
+   "version": "0.0.1",
+   "maintainer": [ "Artur Zakirov <artur.zakirov@adjust.com>", "Manuel Kniep <rapimo@adjust.com>" ],
```
Comment (za-arthur) [inline META.json]:
```suggestion
```

File: META.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+   "license": {
+      "PostgreSQL": "http://www.postgresql.org/about/licence"
+   },
+   "provides": { 
+      "lock_poo": {
+         "file": "sql/pg_lock_pool.sql",
+         "version": "0.0.1",
+         "abstract": "Integers represented in base62"
```
Comment (rapimo) [inline META.json]: copy and paste error maybe?

Comment (PerikAdjust) [reply META.json]: yes thanks

**Review by za-arthur — APPROVED** [review]

**Pull request merged**